src/views/privacyPolicy.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import dwellinglyLogoMobile from "../assets/images/dwellingly_logo_white.png";
+import { Link } from "react-router-dom";
+
+
+
+const PrivacyPolicy = () => {
+
+return(
+  <div className-="privacy-policy__container">
+    <div className = "privacy-policy__mobile__header">
+      <img className= "privacy-policy__mobile__logo" src={dwellinglyLogoMobile}></img>
+    </div>
+    <div className="privacy-policy__text-container">
+      <h3 className="privacy-policy__heading">Privacy Policy</h3>
+        <p className="privacy-policy__text"> 
+              I'm baby blog taxidermy roof party activated charcoal man bun. 
```

Comment:
"Iceland erryday" 😆 so let's check if we have real text for this, I see that we have some in the mockup but I'm not sure if that's just generic privacy policy stuff